Profiles - ZIB 2017/nl-core-patient.xml

```diff
@@ -1,959 +1,1002 @@
+      <alias value="Geslacht" />
+      <mapping>
+        <identity value="hcim-patient-v2.0.1-2015EN" />
+        <map value="NL-CM:0.1.9" />
+        <comment value="Gender. Use FHIR Core ConceptMap for HCIM mapping: http://nictiz.nl/fhir/ConceptMap/GeslachtCodelijst-to-administrative-gender" />
+      </mapping>
+      <mapping>
+        <identity value="hcim-patient-v3.0-2016EN" />
+        <map value="NL-CM:0.1.9" />
+        <comment value="Gender. Use FHIR Core ConceptMap for HCIM mapping: http://nictiz.nl/fhir/ConceptMap/GeslachtCodelijst-to-administrative-gender" />
+      </mapping>
+      <mapping>
+        <identity value="hcim-patient-v3.1-2017EN" />
+        <map value="NL-CM:0.1.9" />
+        <comment value="Gender. Use FHIR Core ConceptMap for HCIM mapping: http://nictiz.nl/fhir/ConceptMap/GeslachtCodelijst-to-AdministrativeGender" />
+      </mapping>
+    </element>
+    <element id="Patient.gender.extension">
+      <path value="Patient.gender.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Patient.gender.extension:GeslachtCodelijst">
+      <path value="Patient.gender.extension" />
+      <sliceName value="GeslachtCodelijst" />
+      <short value="Gender" />
+      <definition value="Patient’s administrative gender." />
+      <alias value="Geslacht" />
+      <max value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/code-specification" />
+      </type>
+    </element>
+    <element id="Patient.gender.extension:GeslachtCodelijst.valueCodeableConcept:valueCodeableConcept">
+      <path value="Patient.gender.extension.valueCodeableConcept" />
+      <sliceName value="valueCodeableConcept" />
+      <binding>
+        <strength value="extensible" />
+        <description value="GeslachtCodelijst" />
+        <valueSetReference>
+          <reference value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.0.1.1--20171231000000" />
+          <display value="GeslachtCodelijst" />
+        </valueSetReference>
+      </binding>
+    </element>
+    <element id="Patient.birthDate">
+      <path value="Patient.birthDate" />
+      <short value="Patients date of birth. The date of birth is mandatory for a patient. An incomplete date, e.g. only year or only year + month, is permitted." />
+      <alias value="Geboortedatum" />
+      <mapping>
+        <identity value="hcim-patient-v2.0.1-2015EN" />
+        <map value="NL-CM:0.1.10" />
+        <comment value="DateOfBirth" />
+      </mapping>
+      <mapping>
+        <identity value="hcim-patient-v3.0-2016EN" />
+        <map value="NL-CM:0.1.10" />
+        <comment value="DateOfBirth" />
+      </mapping>
+      <mapping>
+        <identity value="hcim-patient-v3.1-2017EN" />
+      
```

Comment:
Don't think that this belongs in this PR?

Reply:
I don't know what happened here. There were conflicts and I resolved merged develop into this branch. It should equal nl-core-patient from develop plus zib-lifestance extension. Can you confirm?